crates/macro-support/src/lib.rs

```diff
@@ -24,7 +24,40 @@ mod parser;
 /// Takes the parsed input from a `#[wasm_bindgen]` macro and returns the generated bindings
 pub fn expand(attr: TokenStream, input: TokenStream) -> Result<TokenStream, Diagnostic> {
     parser::reset_attrs_used();
+    // if struct is encountered, add `derive` attribute and let everything happen there (workaround
+    // to help parsing cfg_attr correctly).
     let item = syn::parse2::<syn::Item>(input)?;
+    if let syn::Item::Struct(mut s) = item {
+        s.attrs.insert(
+            0,
+            syn::Attribute {
+                pound_token: Default::default(),
+                style: syn::AttrStyle::Outer,
+                bracket_token: Default::default(),
+                meta: syn::parse_quote! {
+                    derive(wasm_bindgen::prelude::BindgenedStruct)
```

Comment:
We have to parse the attributes to extract the `#[wasm_bindgen(wasm_bindgen = <path>)]` attribute and use it here.